osm2streets-js/src/lib.rs

```diff
@@ -60,4 +63,41 @@ impl JsStreetNetwork {
         let road_network: streets::RoadNetwork = self.inner.clone().into();
         road_network.to_dot()
     }
+
+    #[wasm_bindgen(js_name = getDebugSteps)]
+    pub fn get_debug_steps(&self) -> Vec<JsValue> {
+        // TODO Figure out how to borrow from the RefCell instead of cloning
+        self.inner
+            .debug_steps
+            .borrow()
+            .iter()
+            .map(|x| JsValue::from(JsDebugStreets { inner: x.clone() }))
+            .collect()
+    }
+}
+
+#[wasm_bindgen]
+pub struct JsDebugStreets {
+    inner: street_network::DebugStreets,
+}
+
+#[wasm_bindgen]
+impl JsDebugStreets {
```

Comment:
It's also a bit awkward to have to write a "wrapper struct" with all of these wrapper functions. But it might be mostly unavoidable, or unavoidable without some macros or something.